charts/open-notificaties/templates/configmap.yaml

```
@@ -12,6 +12,8 @@ data:
   DB_NAME: {{ .Values.settings.database.name | toString | quote }}
   CACHE_DEFAULT: {{ .Values.settings.cache.default | toString | quote }}
   CACHE_AXES: {{ .Values.settings.cache.axes | toString | quote }}
+  DEBUG: {{ .Values.settings.debug | toString | quote }}
```

I would use the string values `True` and `False` as described in the manual like this:

```suggestion
  DEBUG: {{ if .Values.settings.debug }}"True"{{ else }}"False"{{ end }}
```